packages/babel-polyfill/package.json

```
@@ -12,6 +12,6 @@
   "main": "lib/index.js",
   "dependencies": {
     "core-js": "^2.5.7",
-    "regenerator-runtime": "^0.12.0"
+    "regenerator-runtime": "^0.13.1"
```

```suggestion
    "regenerator-runtime": "^0.13.2"
```
Version 0.13.2 includes https://github.com/facebook/regenerator/pull/369, which defines `regeneratorRuntime` globally without needing to detect the global object, using [these two techniques](https://github.com/facebook/regenerator/blob/9c90408393bc8e4efc3b715d9e08a80c5f8367b9/packages/regenerator-runtime/runtime.js#L713-L726).